src/commands/synthetics/run-tests-lib.ts

```diff
@@ -68,6 +68,13 @@ export const executeTests = async (
     }
   }
 
+  if (config.mobileApplicationVersion && config.mobileApplicationVersionFilePath) {
+    throw new CiError(
+      'INVALID_CONFIG',
+      'Both mobileApplicationVersion and mobileApplicationVersionFilePath cannot be set at the same time'
+    )
+  }
+
```

Comment:
from reading the summaries from the RFC and i think you also mentioned something about it @Drarig29, think these should be mutually exclusive 🤔 
Also what do you think about where i've put this logic? Is there a better place for it?